docs/langref/relay_expr.rst

```diff
@@ -330,6 +330,51 @@ dependency on the other:
 
 See :py:class:`~tvm.relay.expr.Let` for its definition and documentation.
 
+Graph Bindings
+~~~~~~~~~~~~~~
+
+A :code:`let` binding creates a named variable that is bound to the given value
+and scoped to the subsequent expression. By contrast, a graph binding allows for
+explicitly constructing dataflow graphs in a Relay program by binding an expression
+(graph node) directly to a temporary variable, which is not scoped. This has the
+semantics of substituting the expression wherever the variable appears, even though
+the graph node will only be evaluated once by the compiled program.
+
+These bindings allow for a style of programming that corresponds to that already
+employed by NNVM and other dataflow graph-based input formats. The fact that the variables
+are not scoped offers some flexibility in evaluation order compared to :code:`let`
+bindings, though this can also introduce some ambiguity in programs (`this developer
+page<https://docs.tvm.ai/dev/relay_intro.html>`__ includes more detailed discussion of
```

Comment:
we can use reStructuredText label and ref to directly refer to docs in the same project. See
http://docutils.sourceforge.net/docs/user/rst/quickref.html#hyperlink-targets internal targets

Reply:
Thanks, I'll change it